### DISCUSSION TITLE
What steps should I follow for pull request in this case.

### DISCUSSION THREAD

**Discussion by redxz5** [python-geeks/Automation-scripts#967] [Q&A] [answered]
I have an issue in my pull request (#962) regarding issue #956.
Should I raise a new issue or wait till it is solved?
Are there any other steps I can or should take in this case?

**Comment by pawangeek** [comment, marked as answer]
Well, please raise a Issue first, describing what you want to add in repo, followed by a pull request, 
And the linter issue is solved :)
